orienteer-core/src/main/java/org/orienteer/core/component/structuretable/StructureTableCommandsToolbar.java

```diff
@@ -17,12 +19,14 @@
 {
 	private static final long serialVersionUID = 1L;
 	private RepeatingView commands;
+	private MethodsView methods;
 
 	public StructureTableCommandsToolbar(StructureTable<P, ?> table)
 	{
 		super(table);
         commands = new RepeatingView("commands");
         add(commands);
+		methods = new MethodsView(commands, getModel(),MethodPlace.STRUCTURE_TABLE);
```

Comment:
Is it possible to incorporate into "commands" below as we discussed? Or it's needed as intermediate solution?